src/components/Settings/SettingsPlex.tsx

```diff
@@ -52,12 +62,143 @@ interface SyncStatus {
   libraries: Library[];
 }
 
+interface PresetServerDisplay {
+  name: string;
+  ssl: boolean;
+  uri: string;
+  address: string;
+  host: string;
+  port: number;
+  local: boolean;
+}
 interface SettingsPlexProps {
   onComplete?: () => void;
 }
 
+function PlexPresetOption(props: { key: string; server: PresetServerDisplay }) {
```

Comment:
Use an arrow function to match coding style. You can also deconstruct the props.

```suggestion
const PlexPresetOption = ({ key, server }: { key: string; server: PresetServerDisplay }) => {
```